common/store.ts

```diff
@@ -107,6 +118,7 @@ const configureStore = () => {
           customNodes: state.config.customNodes,
           customNetworks: state.config.customNetworks
         },
+        transaction: state.transaction,
```

Comment:
Why keep transaction state in LS?

Reply:
This is the workaround for grabbing gasPrice from state

Reply:
There might be some privacy issues here if the rest of the transaction data is kept in state. I'd rather keep only gasPrice in state